packages/webpack-config/e2e/custom-homepage/package.json

```diff
@@ -9,7 +9,7 @@
     "react-native-web": "0.17.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9"
+    "@babel/core": "^7.16.0"
```

Comment:
This does not appear to align with any known version of Expo SDK, please revert https://github.com/expo/expo/blob/98255a180d4a4dc4dc39bb54449caaa241df7c58/templates/expo-template-bare-minimum/package.json#L22

Reply:
Yeah, sorry for that, it looks like I have messed up the syncs. 🤦‍♂️